ten_quick_tips_live_coding.md

```diff
@@ -53,46 +53,46 @@ university courses involving the teaching of programming or related techniques e
 
 There is a limited body of research on the effectiveness of Live coding in programming education.
 Most studies focus on non-participatory live coding,
-demonstrating the programming process and opposing this to the use of static code on slides
-(https://doi.org/10.1145/3279720.3279725 and references therein, as well as https://dl.acm.org/doi/10.1145/3373165.3373182).
-So far, the technique show that live-coding is as good as if not better than static code examples.
+demonstrating the programming process and contrasting this with the use of static code on slides
+(see https://doi.org/10.1145/3279720.3279725 and references therein, as well as https://dl.acm.org/doi/10.1145/3373165.3373182).
+So far, the technique show that live-coding is as good as if not better than using static code examples.
 
 The ten quick tips described below are aimed at those interested in applying the technique in to their own teaching.
-These complement the "Ten quick tips for delivering programming lessons"
+They are meant to complement the "Ten quick tips for delivering programming lessons"
 https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007433.
```

Comment:
Time to start assembling the BibTeX file?

Reply:
Next iteration...